deploy/osd-must-gather-operator/01-osd-devaccess.Role.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-devaccess
+  namespace: openshift-must-gather-operator
+rules:
+- apiGroups:
+  - v1
```

Comment:
```suggestion
  - ""
```

Reply:
haha Rick suggested I change this :) I'll change it back

Reply:
"v1" is the apiGroup version, it doesn't get put into roles.  The value of `apiGroups` is an array of apiGroup names.